src/components/portrait/portrait-initials.component.js

```diff
@@ -37,13 +36,10 @@ class PortraitInitials extends React.Component {
 
     let canvas = Browser.getDocument().createElement('canvas');
     let context = canvas.getContext('2d');
+
     let { dimensions } = getSizeParams(theme, size);
 
-    if (!isClassic(theme)) {
-      // For non-Classic themes, reduce the size by 2 pixels to
-      // allow for the CSS border around the canvas <img> element.
-      dimensions -= 2;
-    }
+    dimensions -= 2;
```

Comment:
you can move to to the line above and do

`let { dimensions } = getSizeParams(theme, size) - 2;`